site/content/user-guide/amqp-adapter.md

```diff
@@ -118,6 +118,10 @@ Publish some JSON data for device `4711`:
 
 Notice that we only supplied a new value for the message address, leaving the other default values.
 
+Publish some JSON data for device `4711` using a client certificate for authentication:
+
+    ~/hono/cli/target$ java -jar hono-cli-*-exec.jar --spring.profiles.active=amqp-adapter-cli --hono.client.port=5671 --hono.client.certPath=config/hono-demo-certs-jar/device-4711-cert.pem --hono.client.keyPath=config/hono-demo-certs-jar/device-4711-key.pem --hono.client.trustStorePath=config/hono-demo-certs-jar/trusted-certs.pem --hono.client.hostnameVerificationRequired=false
```

Comment:
is there a reason why the `client-cert` profile is not used here?

Reply:
Like in the other examples in the User Guide of the AMQP adapter, the relevant configuration is given as arguments. I just could not resist to add a profile for client certificates to the application.yml. I would rather remove the profile `client-cert`, than having an example which is implicit as : `~/hono/cli/target$ java -jar hono-cli-*-exec.jar --spring.profiles.active=client-cert`.

Reply:
I agree, in particular since the profile is not really an *add-on* like the *ssl* profile. So, can you please remove the *client-cert* profile then?

Reply:
Done